bin/upgrade-db

```diff
@@ -0,0 +1,84 @@
+#! /usr/bin/env python
+
+import sqlite3
+
+
+def main() -> None:
+    import argparse
+
+    parser = argparse.ArgumentParser(description="Upgrade an existing database\
+            file to version 3")
+    parser.add_argument("dbfiles", nargs="+", type=str,
+                        help="database file(s) to read")
+    parser.add_argument("--suffix", type=str,
+                        help="""a suffix to append to the filename of the
+                        newly upgraded database file""")
+
+    args = parser.parse_args()
+
+    if not args.suffix:
+        args.suffix = "_upgrade"
```

Comment:
```suggestion
    suffix = args.suffix or "_upgrade"
```

Reply:
Implemented. 
Python is a strange language sometimes...